cypress/integration/chi-vue/expansion-panel.spec.js

```diff
@@ -0,0 +1,253 @@
+const EXPANSION_PANEL_DATA_CY = {
+  BASE: {
+    EXPANSION_PANEL_FIRST: '[data-cy="base-expansion-panel-1"]',
+    EXPANSION_PANEL_SECOND: '[data-cy="base-expansion-panel-2"]',
+    EXPANSION_PANEL_THIRD: '[data-cy="base-expansion-panel-3"]',
+    NEXT_BUTTON: `[data-chi-epanel-action='next']`,
+    PREVIOUS_BUTTON: `[data-chi-epanel-action='previous']`,
+    CHANGE_BUTTON: `[data-chi-epanel-action='change']`
+  },
+  STATE: {
+    EXPANSION_PANEL_DONE: '[data-cy="state-expansion-panel-done"]',
+    EXPANSION_PANEL_ACTIVE: '[data-cy="state-expansion-panel-active"]',
+    EXPANSION_PANEL_PENDING: '[data-cy="state-expansion-panel-pending"]',
+  },
+  NUMBER: {
+    NUMBERED_EXPANSION_PANEL: '[data-cy="number-expansion-panel-numbered"]',
+    NON_NUMBERED_EXPANSION_PANEL: '[data-cy="number-expansion-panel-non-numbered"]'
+  },
+  BORDER: {
+    BORDERED_EXPANSION_PANEL: '[data-cy="border-expansion-panel-bordered"]',
+    NON_BORDERED_EXPANSION_PANEL: '[data-cy="border-expansion-panel-non-bordered"]',
+  },
+  SLOT: {
+    ACTIVE_EXPANSION_PANEL: `[data-cy="slot-expansion-panel-active"]`,
+    DONE_EXPANSION_PANEL: `[data-cy="slot-expansion-panel-done"]`
+  }
+};
+const ACTIVE_CLASS = '-active';
+const DONE_CLASS = '-done';
+const PENDING_CLASS = '-pending';
+
+const NUMBER_CLASS = 'chi-epanel__number';
+
+const BORDER_CLASS = '-bordered';
+
+const SLOT_CLASS_ACTIVE = 'chi-epanel__content-active';
+const SLOT_CLASS_DONE = 'done-slot';
+const SLOT_CLASS_FOOTER = 'chi-epanel__footer';
+
+describe('Expansion Panel', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/expansion-panel.html');
+  });
+
+  describe('Base', () => {
+    it('Should check if the first expansion panel is active', () => {
+      cy.get(`${EXPANSION_PANEL_DATA_CY.BASE.EXPANSION_PANEL_FIRST}`)
+        .first()
+        .should('have.class', `${ACTIVE_CLASS}`);
+    });
+
+    it('Should check if clicking Continue opens the next expansion panel and closes the current one', () => {
+      // Given - first panel is active/open and second panel is collapsed/pending
+      cy.get(EXPANSION_PANEL_DATA_CY.BASE.EXPANSION_PANEL_FIRST)
```

Comment:
We need to use aliases, like:
`.as('firstExpansionPanel')`

and then

`.get('@firstExpansionPanel')`